### PR TITLE
Housing Form

### PR DESCRIPTION
**Pull request by SaraVieira**
NOT FINISHED

### CODE REVIEW

**Comment by SaraVieira** [comment]
needs translation and I need some help with the CMS :/

**Comment by SaraVieira** [comment]
Added some translations but they are fropm google translate, pls correct if needed

**Comment by molszanski** [comment]
@SaraVieira <3 will help with the backend in next few hours

File: src/components/add-ticket/AddTicketForm.tsx [hunk 1]
```diff
@@ -56,6 +57,11 @@ export const AddTicketForm = () => {
       active: tag === TagConstIds.transport,
       onClick: () => setTag(TagConstIds.transport),
     },
+    {
+      name: i18n.addTicket.wizard.formNameHousing,
+      active: tag === TagConstIds.housing,
+      onClick: () => setTag(TagConstIds.housing),
```
Comment (PokerTuna) [inline src/components/add-ticket/AddTicketForm.tsx:63]: while ur here, could you please adjust font size on mobile to make it bigger and possibly wrap elements in second picker?

Screenshot 2022-03-10 at 13 26 10

File: src/components/add-ticket/forms/FormNeedTransport.tsx [hunk 2]
```diff
@@ -332,36 +272,7 @@ export const FormNeedTransport = () => {
...
-                py-1 border border-transparent shadow-sm text-sm
-                font-medium text-white bg-green-600
-                focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500
-                text-white cursor-default"
-                >
-                  {translations["pages"]["add-ticket"]["request-added"]}
-                </div>
-              </div>
-            ) : null}
-          </div>
```
Comment (PokerTuna) [inline src/components/add-ticket/forms/FormNeedTransport.tsx:364]: thanks for refactor here <3 lovely

File: src/components/add-ticket/forms/FormNeedTransport.tsx [hunk 3]
```diff
@@ -332,36 +272,7 @@ export const FormNeedTransport = () => {
...
+          <FormFeedback mutation={addTicketMutation} />
```
Comment (PokerTuna) [inline src/components/add-ticket/forms/FormNeedTransport.tsx:275]: yassss! thank you!

File: src/components/add-ticket/forms/hooks.ts [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+import dayjs from "dayjs"
+import { useMutation } from "react-query"
+import {
+  NeedHousingPostData,
+  NeedTripPostData,
+} from "../../../services/ticket.type"
+import { TagConstIds } from "../../../services/types.tag"
+import { getRootContainer } from "../../../services/_root-container"
+
+export const useAddTransportTicket = ({ onSuccess }) => {
```
Comment (PokerTuna) [inline src/components/add-ticket/forms/hooks.ts:12]: <3

File: src/components/add-ticket/forms/hooks.ts [hunk 5]
```diff
@@ -0,0 +1,122 @@
+import { useQuery } from "@chakra-ui/react"
```
Comment (PokerTuna) [inline src/components/add-ticket/forms/hooks.ts:1]: looks like this can be removed

**Review by PokerTuna — APPROVED** [review]

File: src/components/add-ticket/AddTicketForm.tsx (hunk 1 above)
Comment (SaraVieira) [reply src/components/add-ticket/AddTicketForm.tsx:63]: How do you feel about this?

CleanShot 2022-03-10 at 14 00 16@2x


Would you still like them to wrap?

File: src/components/add-ticket/forms/hooks.ts (hunk 5 above)
Comment (SaraVieira) [reply src/components/add-ticket/forms/hooks.ts]: Done :)

File: src/components/add-ticket/AddTicketForm.tsx (hunk 1 above)
Comment (PokerTuna) [reply src/components/add-ticket/AddTicketForm.tsx:63]: Depends how it looks on UA, but here looks ok!

File: src/components/add-ticket/forms/FormNeedHousing.tsx [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+      }, 1000)
+    },
+  })
+
+  const mappedLocationTags = useMemo(() => {
+    return [
+      {
+        value: 99,
+        label: "Anywhere",
+      },
```
[the remark this answers is not in the document]
Comment (molszanski) [reply src/components/add-ticket/forms/FormNeedHousing.tsx:51]: already working on it #79

**Review by molszanski — COMMENTED** [review]
Wow wow wow <3

File: src/components/add-ticket/forms/FormNeedHousing.tsx (hunk 6 above)
[the remark this answers is not in the document]
Comment (molszanski) [reply src/components/add-ticket/forms/FormNeedHousing.tsx]: done

**Review by molszanski — APPROVED** [review]
:)

**Pull request merged**